### ISSUE TITLE
UX regression: setfiles progress indicator is now misleading and confusing in fixfiles

### ISSUE THREAD

**Issue by sourcejedi** [SELinuxProject/selinux#45]
I have this bug report written, I likely can't do anything further with it for a few days.

In particular, this issue will be annoying when `fixfiles onboot` is invoked.

"Good" version: policycoreutils-2.5-19.fc25.x86_64:

```
Relabeling / /boot /dev /dev/hugepages /dev/mqueue /dev/pts /dev/shm /home /run /run/user/1000 /run/user/1001 /run/user/42 /sys /sys/fs/pstore /sys/kernel/debug /tmp
100.0%
Cleaning up labels on /tmp
```

"Bad" version: 20161014-199-gd66c54e

* built according to README
* hacked fixfiles script to use ~alan-sysop/obj/sbin/setfiles
* sudo sh -c 'LD_LIBRARY_PATH=~alan-sysop/obj/lib sh ./policycoreutils/scripts/fixfiles restore'

Result: My impression from watching `fatrace`, was that the percentage indicator only applied to the first filesystem `/`. Once fixfiles got to `/home`, it was just printing asterixes.  `/home` can potentially have many more files than `/`.

```
Relabeling / /boot /dev /dev/hugepages /dev/mqueue /dev/pts /dev/shm /home /run /run/user/1000 /run/user/1001 /run/user/42 /sys /sys/fs/pstore /sys/kernel/debug /tmp
100.0%
100.0%
100.0%

*




******************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
*



********************************



```

**Comment by stephensmalley** [comment]
Yes, this is a regression.

**Comment by stephensmalley** [comment]
Fixed by 454768f56d7a941657d800e303994bca086b7546
There is a change in the UX with respect to reporting progress per-filesystem; hopefully that isn't too noisy.
Other, separate issue is likely that we ought to exclude more of these filesystems in fixfiles, as we used to do before introducing seclabel mount option.
